index.html

```diff
@@ -0,0 +1,125 @@
+      <a id="button-main-bar" href="https://nimplay.itch.io/">Jam</a>
+    </div>
+  </nav>
+</header> 
+  <section class="home" id="hero">
+    <div class="home-titles">
+      <h2 id="hello" class="text-orange">"Hello world"</h2>
+      <h1 id="maintitle" class="text-orange">NIMPLAY jam 2023</h1>
+    </div>
+    <div class="home-p-box">
+      <p class="text-gray">Enjoy the challenge of programming a video game in 15 days, 
+        show your skills in game development in our jam</p>
+    </div>
+    <div class="home-button-text">
+      <h2 id="home-date" class="text-gray">2023.09.15(Friday) - 30(Saturday). </h2>
+      <h3 class="text-gray">@NIMPLAY Academy</h3>
+    </div>
+  </div>
+  </section>
+  <section class="program">  
+    <h2 id="program-title" class="text-color">Main Program</h2>  
+    <div id="red-line"></div>
+    <div id="program-card">
+    </div>
+    <button class="card-buttom" aria-label="button">NIMPLAY jam 2023</button>
+    <p id="program-text-footer">SEE THE WHOLE PROGRAM</p>
+  </section>
+  <section class="participants">
+    
+   <div id="speaker-continer">
+    <h2 id="program-title" class="text-gray">Featured Guests</h2>  
+    <div id="red-line2"></div>
+    <div id="static">
+    <div class="speaker-data">
+      <img class="speaker-image" src="./images/Nimrod.jpg" alt="">
+      <div class="speaker-data2">
+      <h2 class="speaker-title">Nimrod Acosta</h2>
+      <p class="speaker-paragraph">Nimplay's CEO and video game developer .</p>
+        <div class="speaker-line"></div>
+      <p class="speaker-paragraph2">Four apps in Google Playstore and 3 years of experience 
+        programming video games, also 10 years like Music Coposition.</p>
+      </div>
+    </div>
+    <div class="speaker-data">
+      <img class="speaker-image" src="./images/speaker_02.png" alt="">
+      <div class="speaker-data2">
+      <h2 class="speaker-title">Nahomi Noh</h2>
+      <p class="speaker-paragraph">Professor in the Video Game University .</p>
+        <div class="speaker-line"></div>
+      <p class="speaker-paragraph2">As the main venue for new media art production in 
+        Korea. Lorem ipsum dolor sit amet consectetur.</p>
+    </div>
+    </div>
+  </div>
+  </div>
+  <button id="speaker-buttom" class="card-buttom2" aria-label="button"> MORE </button>
+  </section>
+  <section class="main-sponsor-continer">    
+    <div id="sponsor-continer">
+     <h2 id="program-title" class="text-color">Sponsors</h2>  
+     <div id="red-line2"></div>    
+     <div class="sponsor"> 
+       <img src="./images/arcadia.png" alt="">  
+       <img src="./images/kcc.png" alt=""> 
+       <img src="./images/mozilla.png" alt="">  
+     </div>
+     <div class="sponsor2"> 
+       <img src="./images/lumen.png" alt="">
+       <img src="./images/line-friends.png" alt="">   
+     </div>
+   </div>  
+   </section>
+   <footer class="footer">
+    <img class="footer-img" src="./images/cck-logo1.png" alt="">
```

Comment:
Please add alternative text to all of your images
<img width="834" alt="Screen Shot 2022-06-23 at 9 08 45" src="https://user-images.githubusercontent.com/82426682/175319525-d75f4516-fed2-46de-86b1-bda3ce70d58d.png">

Reply:
ok I copy!